apps/omg/test/omg/state/utxo_set_test.exs

```diff
@@ -63,6 +63,16 @@ defmodule OMG.State.UtxoSetTest do
                |> UtxoSet.init()
                |> UtxoSet.get_by_inputs(inputs)
     end
+
+    test "ignores db query possible not_founds",
```

Comment:
I'm having difficulties understanding the purpose of this test... a better description would help.